### ISSUE TITLE
Any plans to add a CI integration?

### ISSUE THREAD

**Issue by renjith4** [rubicon-project/prebid-server-java#559]
This is more of a feature request :-) . Not sure whether the need for CI was discussed before in the community. Couldn't find a corresponding issue-ticket and hence raising this. 

I was having a look at the prebid-server-java recently. Appreciate the good work that you all are doing and the efforts taken to open source it to the community. While having a play around, two things which I thought could be beneficial for the project.  

1. Need for a CI
2. Need to have a discussion list or similar documentation

The reason why I would stress for a CI is that the tests on master branch were broken for few days which could have picked up on a CI quicker. In this instance the culprit was a non-UTC date in one of the tests.  Saw the usages of `new Date()` in the code which should be ideally avoided since we have java.time packages from java8 and there are fixed clocks which could be used for tests. 

The reason for a discussion list or similar is to improve the knowledge/feedback on bidders/ adapters when someone is facing an issue.  Maybe adding extra documentation would also help. 

If there is an appetite for these then I could take a look and maybe talk to free services for open source. (for example https://circleci.com/open-source/). If there are already services in place satisfying these requirements then please point me in that direction.

**Comment by bretg** [comment]
@renjith4 - thanks for reaching out, glad you appreciate the team's efforts. Yes, that's certainly a gap -- we'll add CI in the next sprint or two.

As for discussion forums, there are several options:
- a mailing list of PBS adapter owners. If your team has an adapter but isn't on the list, let me know
- issues like this one
- for Prebid.org members (even the free 'community tier' members):
  - Prebid.org Server committee meets every two weeks on Friday
  - Prebid.org has a slack workspace

**Comment by renjith4** [comment]
Thanks for the reply @bretg . It is good to hear that adding a CI is in the pipeline. 

Our team is working on few adapters and it is not ready to be merged yet (few permissions, third party company confidentiality things to be sorted out before that). Could we be added to adapter owners list as we got some questions regarding general architecture?

Secondly we got some questions regarding existing adapters, for example Rubicon. Will open another issue for that as you suggested. 

Feel free to close this issue as you have answered my questions.
